app/ui/src/Pages/NewUser/NewUserUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { CHECK } from 'kwc';
+import { AccessLevel } from 'Graphql/types/globalTypes';
+
+export function verifyUsername(value: string) {
+  return CHECK.getValidationError([
+    CHECK.isFieldNotEmpty(value),
+    CHECK.isLengthAllowed(value, 40),
+    CHECK.matches(
+      value,
+      new RegExp(/^[a-zA-Z0-9._-]*$/g),
+      'The username should contain only alphanumeric, dash (-), underscore (_) and dot (.) characters'
+    ),
+  ]);
+}
+export function verifyPassword(value: string) {
+  return CHECK.getValidationError([
+    CHECK.isFieldNotEmpty(value),
+    CHECK.matches(
+      value,
+      new RegExp(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{6,}$/g
+      ),
+      'The password length cannot be less than 6 characters, should contain at least one digit, one special character (@$!%*?&), one lowercase character and one uppercase character.'
+    ),
```

Comment:
We should split this validation.
```suggestion
    CHECK.matches(value, /.{6,}/, 'Password must contain at least 6 characters'),
    CHECK.matches(
      value,
      new RegExp(
        /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]$/g
      ),
      'The password must contain at least one digit, one special character (@$!%*?&), one lowercase character and one uppercase character.'
    ),
```